static/.eslintrc

```diff
@@ -41,18 +41,39 @@ Do not edit this file in your local project
     "comma-dangle": [2, "never"],
     "comma-spacing": [2, { "before": false, "after": true }],
     "comma-style": [2, "last"],
+    "computed-property-spacing": [2, "never"],
```

Comment:
Does `standard/computed-property-even-spacing` only cover newlines or didn't we install it properly?
https://github.com/upfrontIO/apply-guides/pull/27/files#diff-2072934b5106b60a8a6ffc4737a34a21R194

We depend on `eslint-plugin-standard`, which isn't installed in the editor.